src/Components/Pages/CartPage.jsx

```diff
@@ -21,31 +24,28 @@ function CartPage() {
           <div className="w-2/4">
             <h6>Total</h6>
           </div>
+          <div className="w-2/4">
+            <h6>Actions</h6>
+          </div>
         </div>
-        <hr className=" opacity-[0.2]" />
+
+        <hr className="opacity-[0.2]" />
       </div>
-      <div className=" flex mt-6 gap-7 mx-auto container">
-        <CartItemCard
-          image={"/assets/images/dauda.webp"}
-          quality={1}
-          total={"499,999"}
-        />
-        <div className="gap-2">
-          <button className="bg-[#ddd8d8] py-2 rounded-full px-5">
-            <IoAdd />
-          </button>
-          <button className="bg-[#ddd8d8] py-2 rounded-full px-5">
-            <IoRemove />
-          </button>
-        </div>
-        <a
-          href=""
-          className="hover-underline"
-          onClick={() => removeFromCart(product.id)}
-        >
-          Remove
-        </a>
+
+      <div className="mt-6 gap-7 mx-auto container">
+        {items?.map((item, index) => (
+          <React.Fragment key={index}>
```

Comment:
Using the `item.id` if id exists for each item might be a better `key`. If an `id` doesn't exist. You may want to consider prepending a piece of text to generate unique keys e.g `cart-item-${index}`

Reply:
Noted,thank you sir.